File: samplejava/src/main/java/mx/leo/sample/easyrecycler/viewholder/SampleItemViewHolder.java
package mx.leo.sample.easyrecycler.viewholder;

import android.view.View;
import android.widget.ImageView;

import com.bumptech.glide.Glide;

import org.jetbrains.annotations.NotNull;

import butterknife.BindView;
import butterknife.ButterKnife;
import mx.leo.easyrecycler.viewholder.EasyItemViewHolder;
import mx.leo.sample.easyrecycler.R;
import mx.leo.sample.easyrecycler.model.Language;

/**
 * Created by Braulio on 12/12/2016.
 */

public class SampleItemViewHolder extends EasyItemViewHolder {

    @BindView(R.id.sample_image_view) ImageView sampleImageView;

    public SampleItemViewHolder(@NotNull View view) {
        super(view);
        ButterKnife.bind(this, view);
    }

    public void bindItem(Language language) {
        Glide.with(sampleImageView.getContext()).load(language.getPhoto()).into(sampleImageView);
    }
}

File: samplekotlin/src/main/java/brauliomendez/com/kotlin/viewholder/SampleItemViewHolder.kt
package brauliomendez.com.kotlin.viewholder

import android.view.View
import brauliomendez.com.kotlin.model.Language
import com.bumptech.glide.Glide
import kotlinx.android.synthetic.main.item_layout.view.*
import mx.leo.easyrecycler.viewholder.EasyItemViewHolder

/**
 * Created by Braulio on 26/12/2016.
 */
class SampleItemViewHolder(view : View) : EasyItemViewHolder(view) {

    fun bindItem(language : Language) {
        Glide.with(itemView.sample_image_view.context).load(language.image).into(itemView.sample_image_view)
    }
}